### PR TITLE
Adding the sign up and mobile sign up page, verification page,upload page

### LINKED ISSUE #23: build the sign-up page
- build the mobile screen for the sign-up page.
- build the desktop screen for the sign-up page.

### PR DESCRIPTION
**Pull request by Preciousglows**
Resolves #23

### CODE REVIEW

**Comment by Preciousglows** [comment]
Resolves #41 

These are the screens for the verifcation page. @jeremiahlateef @Festusali

File: frontend-development-aboutus-#39/.vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (Festusali) [inline frontend-development-aboutus-#39/.vscode/settings.json:1]: Consider adding your local vscode settings to gitignore list

**Review by Festusali — CHANGES_REQUESTED** [review]
Point your pull request to frontend development branch rather than the main branch.

The code base and files in this pull request doesn't have all the necessary code base files required for our project to run on a Django server

**Comment by Preciousglows** [comment]
@Festusali I have changed the branch

**Comment by jeremiahlateef** [comment]
Don't push to that branch @Preciousglows. I already have files there. It will override it. Create your own branch and push it there. @Festusali, please wait for her to change her brach before accepting the pull request. It will override everything I have put there.

**Comment by jeremiahlateef** [comment]
I have changed the branch to #41.

**Comment by jeremiahlateef** [comment]
@Preciousglows There are some significant changes I want you to make to your code before I accept it. 
- Don't use generic names for the class. Don't use names like **form** or **input** to name your classes.
- Don't use a tag without using adding a class to it. Don't use **span**, **form**, **h2** without adding a class to it. 
- I've used some of these classes on the template, so when you use a class I have already used, it affects all the elements with that class, the same goes for tags. That's why your footer changed while working on the sign-up page. 
- So, go back to your files and rename your classes. Your codes are not that long. I don't have the time to review and change everything as @iamjooshua also made the same mistakes. 
- Also, if you can, connect your repository to a server like netlify so you can view the live project.

**Pull request merged**